src/sass/main.scss

```diff
@@ -6,14 +6,14 @@ $mintFontsPath: 'fonts/' !default;
 @import "helpers";
```

Comment:
as I mentioned, I did some name changes and moved files around to make structure a bit more logical